src/widgets/upcoming-courses/ui/upcoming-courses.module.scss

```diff
@@ -0,0 +1,26 @@
+.course-title {
+  margin-top: 24px;
+}
+
+.rs-banner {
+  width: 300px;
+  height: 200px;
+
+  @include media-laptop {
+    margin-top: 24px;
```

Comment:
You can use `gap` instead

Reply:
the nearest parent of the image containing `column-2` class which is a global class
 
<img width="1290" alt="Screenshot 2024-10-01 at 18 29 19" src="https://github.com/user-attachments/assets/0586070f-d9f9-4166-96ff-43d5eb1eb47a">

<img width="633" alt="Screenshot 2024-10-01 at 18 28 48" src="https://github.com/user-attachments/assets/12347229-7f79-4e1a-bb7e-0672637427d6">

Reply:
I suppose we can't overwrite global styles, but we can supplement them with our own classes and hang additional properties on them. However, this is up to you, you can resolve the comment